src/ws/login.js

```diff
@@ -406,19 +415,27 @@
             } else {
                 show_form(in_conversation);
                 id("login-error-message").textContent = msg;
-                id("error-group").style.display = "block";
+                id("error-group").style.display = "grid";
```

Comment:
It's wrong to set CSS in the JavaScript (see above). We're already doing currently in the code (which is icky). 

Ideally, when updating code, it would be nice to also make it use best practices instead of just changing it a bit.

(...That said, whenever I touch the login page, I also try not to change much functionally...)